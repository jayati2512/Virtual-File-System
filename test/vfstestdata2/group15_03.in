createvfs VFSImplementationforDMProjectusingCimplementedinUbuntu 1024
createvfs jkl; 1024
mountvfs jkl;
makedir /home/user/ abc 
addfile /home/user/ test.txt
movedir /home/ /gome/
makedir / gome
deletedir /home/
removefile /home/user/test.txt
deletedir /home/
addfile / test.txt /usr/include/stdio.h
addfile / test.txt /usr/include/poll.h
exportfile 
exportfile /test.txt /tmp/output.txt
listfile /test1.txt /tmp/output.txt
listfile /test.txt
listfile /test.txt /tmp/output1.txt
listdir / 1 /tmp/output2.txt
unmountvfs 
unmountvfs jkl;
createvfs iop[ 1025
makedir / home
