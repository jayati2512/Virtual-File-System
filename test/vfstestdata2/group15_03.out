createvfs_FAILURE VFS_LABEL_TOO_LARGE
createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
addfile_FAILURE VFS_INSUFFICIENT_ARGUMENTS
movedir_FAILURE CANNOT_FIND_SPECIFIED_DESTINATIONDIR
makedir_SUCCESS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
removefile_FAILURE CANNOT_FIND_SPECIFIED_FILE
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
addfile_FAILURE FILE_TOO_LARGE
addfile_SUCCESS
exportfile_FAILURE VFS_INSUFFICIENT_ARGUMENTS
exportfile_SUCCESS
listfile_FAILURE SOURCE_FILE_PATH_NOT_FOUND
listfile_FAILURE VFS_INSUFFICIENT_ARGUMENTS
listfile_SUCCESS
listdir_SUCCESS
unmountvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
unmountvfs_SUCCESS
createvfs_FAILURE INVALID_SIZE
VFS_NOT_MOUNTED
